wagtail/contrib/redirects/models.py

```diff
@@ -27,6 +29,25 @@ class Redirect(models.Model):
         on_delete=models.CASCADE
     )
     redirect_link = models.URLField(verbose_name=_("redirect to any URL"), blank=True, max_length=255)
+    automatically_created = models.BooleanField(
+        verbose_name=_("automatically created"),
+        default=False,
+        editable=False,
+    )
+    created_at = models.DateTimeField(verbose_name=_("created at"), auto_now=True, null=True)
```

Comment:
Fix: ``auto_now`` should be ``auto_now_add``

Reply:
@kaedroho Good catch! Do you know off the top of your head if redirect changes are covered by logging? I'm wondering what should happen if an automatically created redirect is manually updated? Should they be unmarked as automatically created at that point? Should there be a checkbox on the edit form perhaps?